docs/efun/system/find_call_out.md

```diff
@@ -5,19 +5,43 @@ title: system / find_call_out
 
 ### NAME
 
-    find_call_out() - find a call out scheduled to be called next
+    find_call_out() - find a call out scheduled to be called
 
 ### SYNOPSIS
 
-    int find_call_out( string func ):
+    int find_call_out( string func ) ;
+    int find_call_out( int handle ) ;
 
 ### DESCRIPTION
 
-    Find  the  first  call  out due to be executed for function 'func', and
-    return the time left. If it is not found, then return -1.
+    Find a call out scheduled to be called either by function name or by
+    handle.
+
+    If the argument is a string, then the first call out due to be executed
+    for function 'func' is found, and the time left in seconds is returned.
+    If no call out is found, then return -1.
+
+    If the argument is an integer, then the call out with handle 'handle'
+    is found, and the time left in seconds is returned. If no call out is
+    found, then return -1.
+
+    This will function to find call outs scheduled by call_out() or
```

Comment:
`This will function to find call outs` could probably be simplified, maybe to `This will find call outs`